src/main/java/com/frcteam3255/components/motors/SN_SparkMax.java

```diff
@@ -0,0 +1,194 @@
+package com.frcteam3255.components.motors;
+
+import com.ctre.phoenix.ErrorCode;
+import com.ctre.phoenix.motorcontrol.DemandType;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.BaseTalonConfiguration;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkMaxPIDController;
+
+public class SN_SparkMax extends CANSparkMax implements SN_MotorInterface{
```

Comment:
I would prefer to call it `SN_CANSparkMax`, since `PWMSparkMax` exists (although I can't think of why we would ever use it).